components/Document/DocumentLineItems.tsx

```diff
@@ -118,7 +118,9 @@ const DocumentLineItems = ({
                 href={
                   isPaper(document) && document.externalUrl
                     ? document.externalUrl
-                    : `https://` + document.doi
+                    : document.doi.startsWith("http")
```

Comment:
There was also an issue where the DOI field on the paper page added an extra `https://`; e.g. `https://https://doi.org/10.1364/OFC.2022.W1H.1`

Reply:
I see. look okay